src/main/drivers/vtx_common.h

```diff
@@ -28,21 +28,25 @@ typedef enum {
 
 struct vtxVTable_s;
 
+typedef struct vtxDeviceCapability_s {
+    uint8_t bandCount;
+    uint8_t channelCount;
+    uint8_t powerCount;
+} vtxDeviceCapability_t;
+
 typedef struct vtxDevice_s {
     const struct vtxVTable_s *vTable;
 
-    uint8_t numBand;
-    uint8_t numChan;
-    uint8_t numPower;
+    vtxDeviceCapability_t capability;
 
-    uint16_t *freqTable;  // Array of [numBand][numChan]
-    char **bandNames;    // char *bandNames[numBand]
-    char **chanNames;    // char *chanNames[numChan]
-    char **powerNames;   // char *powerNames[numPower]
+    uint16_t *freqTable;  // Array of [bandCount][channelCount]
+    char **bandNames;    // char *bandNames[bandCount]
+    char **chanNames;    // char *chanNames[channelCount]
```

Comment:
I am against the use of unnecessary abbreviations, especially when they are inconsistent. Using `chan` rather than `channel` serves no purpose and can cause confusion.
1. When reading code the question arrises "Does `chan` mean something subtly different from `channel`, or is it just an inconsistency?"
2. When writing new code you always have to think "Hmm, is the variable called `chan` or `channel`?" which just slows you down.

Just use `channel` throughout.